mailpoet/lib/Automation/Engine/Storage/AutomationRunStorage.php

```diff
@@ -108,6 +109,26 @@ function(array $subjectData) use ($runData): bool {
     );
   }
 
+  /**
+   * @param Automation $automation
+   * @return int
+   */
+  public function countRunsForAutomationAndSubject(Automation $automation, Subject $subject): int {
```

Comment:
I guess more consistent naming with what we use in repositories is `getCountByAutomationAndSubject` or `getCountForAutomationAndSubject`.

Reply:
979c51c5e